autoload/neoterm/repl/python.vim

```diff
@@ -3,27 +3,37 @@ if !exists('g:neoterm_repl_ipy_magic')
   let g:neoterm_repl_ipy_magic = 0
 end
 
-if !exists('g:neoterm_ipy_cellmark')
-  let g:neoterm_repl_ipy_cellmark = "# %%"
-end
-
-function! neoterm#repl#python#cell()
-    let l:lnum1 = search(g:neoterm_repl_ipy_cellmark, 'Wb') + 1
-    let l:lnum2 = search(g:neoterm_repl_ipy_cellmark, 'W') - 1
-    if l:lnum2 == -1
-      let l:lnum2 = line('$')
+function! neoterm#repl#python#is_valid(value)
+  let l:pyCmd = 0
+  let l:invalid = 0
+  if type(a:value) == v:t_list
+    for i in range(len(a:value))
+      if a:value[i] =~ 'python'
+        let l:pyCmd = 1
+        if a:value[i] == 'ipython'
+          let g:neoterm_repl_python[i] = 'ipython --no-autoindent'
+        endif
+      endif
+      if executable(split(a:value[i])[0]) == 0
+        let l:invalid = 1
+      endif
```

Comment:
In python i would create a function to handle the list item and the string. Do you know if it is possible here?

Reply:
Sorry, I don't understand your intention. What do you mean?

Reply:
Has already cleared up.